tasks/eth/006-ecotone-set-gas-config/README.md

```diff
@@ -0,0 +1,76 @@
+# Mainnet - Ecotone Blobs Gas Configuration
+
+Status: DRAFT
+
+## Objective
+
+This is the playbook for updating the gas config of the `SystemConfig` to be ready to send
+blob batch transactions.
+
+The proposal was:
+- [ ] approved by Token House voting here: TODO 
+- [ ] not vetoed by the Citizens' house here: TODO
+
+Following [internal calculations](https://github.com/ethereum-optimism/protocol-quest/issues/84#issuecomment-1977330679),
+the basefee and blob basefee scalars are updated to
+* `L1BaseFeeScalar: 1353`
+* `BlobBaseFeeScalar: 810945`
+
+These values are optimized for sending 6 blobs per transactions on OP Mainnet.
+
+### Timing
+
+The transaction is scheduled for execution a few minutes after Ecotone activates at
+`Mar 14 00:00:01 UTC`.
+This gas configuration change is necessary to switch the batcher to blob transactions.
+
+## Transaction creation
+
+The [`ecotone-scalar`](https://github.com/ethereum-optimism/optimism/tree/develop/op-chain-ops/cmd/ecotone-scalar)
+encoding tool was used to determine the correct transaction input:
+```
+go run ./op-chain-ops/cmd/ecotone-scalar --scalar=1353 --blob-scalar=810945
+# base fee scalar     : 1353
+# blob base fee scalar: 810945
+# v1 hex encoding  : 0x010000000000000000000000000000000000000000000000000c5fc100000549
+# uint value for the 'scalar' parameter in SystemConfigProxy.setGasConfig():
+452312848583266388373324160190187140051835877600158453279134670513164518729
+```
+
+This encoding follows the [spec change for Ecotone fee scalars](https://github.com/ethereum-optimism/specs/blob/11099e9908bb7bfa640d73b2a3a2349bef9ab7a1/specs/protocol/system_config.md#scalars).
+Notably, in version 1 of the scalar encoding format, the `overhead` is set to 0 and the old `scalar` field
+now encodes the _base fee scalar_ as well as the _blob base fee scalar_ in a packed format.
+The first byte of the `scalar` field denotes the version `0x01`.
+
+The transaction was created in the root directory with
+
+```
+just add-transaction tasks/eth/006-ecotone-set-gas-config/inputs/op.json 0x229047fed2591dbec1eF1118d64F7aF3dB9EB290 'setGasConfig(uint256,uint256)' 0 0x010000000000000000000000000000000000000000000000000c5fc100000549
+```
+
+## Signing and execution
+
+Please see the signing and execution instructions in [SINGLE.md](../../../SINGLE.md).
+
+## Validations
+
+### State 
+
+On the "State" tab, you can verify that the following two state changes occured on the `SystemConfigProxy` at `0x229047fed2591dbec1ef1118d64f7af3db9eb290`:
+
+* `overhead` (key `0x65`) changed from `0xbc = 188` to `0`
+* `scalar` (key `0x66`) changed
+  * from `0x00000000000000000000000000000000000000000000000000000000000a6fe0 = 684000`
+  * to `0x010000000000000000000000000000000000000000000000000c5fc100000549`
+    `= 452312848583266388373324160190187140051835877600158453279134670513164518729`
```

Comment:
formatting feels off, should it be this?
```suggestion
  * to `0x010000000000000000000000000000000000000000000000000c5fc100000549 = 452312848583266388373324160190187140051835877600158453279134670513164518729`
```

Reply:
Also, what is the reason for showing the decimal representation of that number? It would be more useful to show how to decode the hex value into the actual config being used.

Reply:
I just showed the decimal for completion, because both the script and state diff show both. Happy to remove the decimal representation altogether.
I can add the decoding command as well.

Reply:
Yes to replacing decimals with decoding command. TY